libs/sdk-ui-kit/src/assets/calpine.svg

```diff
@@ -0,0 +1,25 @@
+<svg width="138" height="34" viewBox="0 0 138 34" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Do we have permission to use the logos in our SDK?

Reply:
Hi @no23reason , I'm not use which permission we should have. Could you please provide more detail?
Currently, these icons are only used for the GoodData pricing dialog, this dialog will be used by the Freemium apps (AD, KD, Home, Data source and MSF ...) for Free users.

Reply:
I meant do the companies the logos belong to allow us to use them this way?